src/tests/btree_helpers/btree_test_helper.hpp

```diff
@@ -92,15 +92,44 @@ struct BtreeTestHelper {
             for (std::size_t i = 0; i < n_fibers; ++i) {
                 const auto start_range = i * chunk_size;
                 const auto end_range = start_range + ((i == n_fibers - 1) ? last_chunk_size : chunk_size);
-                iomanager.run_on_forget(m_fibers[i], [this, start_range, end_range, &test_count]() {
-                    for (uint32_t i = start_range; i < end_range; i++) {
-                        put(i, btree_put_type::INSERT);
-                    }
-                    {
-                        std::unique_lock lg(m_test_done_mtx);
-                        if (--test_count == 0) { m_test_done_cv.notify_one(); }
-                    }
-                });
+                auto fiber_id = i;
+                iomanager.run_on_forget(
```

Comment:
Could you please move/copy this printing the progress to a function if possible and use it on concurrent test ops too. Today we dont have any visibility of whether its stuck or in progress.

Reply:
both of them have progress printing but they are slightly different (in terms of printing) because preload doesn't have deadline (time to stop).